Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["graphprotocol", "gossip-network", "sdk", "waku", "p2p"]
 categories = ["network-programming", "web-programming::http-client"]
 
 [dependencies]
-waku = { version = "=0.1.1", package = "waku-bindings" }
+waku = { git = "https://github.com/waku-org/waku-rust-bindings", branch = "update-waku-version", package = "waku-bindings" }
```

Comment:
I think we can directly use version [0.3.0](https://crates.io/crates/waku-bindings) now